pkgs/desktops/plasma-5/kinfocenter/default.nix

```diff
@@ -99,6 +99,6 @@ mkDerivation {
   # the same directory, while it is actually located in a completely different
   # store path
   preFixup = ''
-    ln -sf ${lib.getExe systemsettings} $out/bin/kinfocenter
+    ln -sf ${systemsettings}/bin/systemsettings $out/bin/kinfocenter
```

Comment:
Just curious: does `getExe` break anything here?

Reply:
Yes, it picks `systemsettings5` which is a symlink that ends up double wrapped.

Reply:
Then we probably should do `meta.mainProgram = "systemsettings";`

Reply:
What we should really do is to fix wrapQtAppsHook so it doesn't touch symlinks.

Reply:
Cool, `About this System` menu entry works fine now (before, it just opened System Settings home screen)